utils/httpUtils.ts

```diff
@@ -72,3 +71,114 @@ export const notFound = (message: string) => {
     message: message,
   });
 };
+
+export const getFeatureFlagsFromRequest = context => {
+  return context.request.headers.get("x-should-read-from-nextgen");
+};
+
+export const shouldReadFromNextGen = async context => {
+  let shouldReadFromNextGen = getFeatureFlagsFromRequest(context);
+  if (shouldReadFromNextGen === true || shouldReadFromNextGen === "true" || shouldReadFromNextGen === "True") {
```

Comment:
lol do you mind aadding a comment as to why this is?

Reply:
I actually have no idea @saq7 do you have context on this?

Reply:
The `if` statment? That was to cover boolean conversions, etc. Basically a hedge at the time against whatever the frontend sent